src/docs/sdk/event-payloads/properties/sampled.mdx

```diff
@@ -0,0 +1,9 @@
+`sampled`:
+
+: _Required_. Determines if the the Span should be send, if `!== true` it will be sampled and therefore not sent.
```

Comment:
Yeah, I would say it's up to the SDK to decide, relay shouldn't reject this.

Reply:
Since now we are finally documenting what the protocol is, what is the role of sending `sampled: true` or `sampled: false` and marking it as **required** field?

Reply:
@HazAT I believe `sampled` should not be part of the JSON payload.